src/nginx/habitat/hooks/health_check

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+
+# Health Check for oc_erchef
+
+# default return code is 0
+rc=0
+
+{{pkgPathFor "core/curl"}}/bin/curl -k -X GET -sS --fail --max-time 2 https://localhost/_status
```

Comment:
I wonder if this is the best health_check for nginx.  This will fail if anything is down on the erchef side too.